zingolib/src/wallet/transaction_context.rs

```diff
@@ -218,6 +203,11 @@ pub mod decrypt_transaction {
                 for (n, vout) in t_bundle.vout.iter().enumerate() {
                     if let Some(taddr) = vout.recipient_address() {
                         let output_taddr = address_from_pubkeyhash(&self.config, taddr);
+                        // If there's at least one output in this bundle that does *NOT* belong to the
+                        // Creator, then the transaction is defined as "is_outgoing"
+                        if !taddrs_set.contains(&output_taddr) {
+                            *is_outgoing_transaction = true;
+                        }
```

Comment:
hmm what about the case where a different capability has sent to us and another recipient. in that case we would call this function and think we created it?